src/commcare_cloud/ansible/roles/nginx/tasks/install.yml

```diff
@@ -75,3 +75,36 @@
   import_tasks: roles/commcarehq/tasks/www_log_dir.yml
 
 - set_fact: nginx_installed=True
+
+- name: Create dynamic config change script
+  become: yes
+  copy:
+    src: dynamic_cfg_change_nginx
+    dest: "/usr/local/bin"
+    owner: "root"
+    group: "root"
+    mode: 0755
+  when: dynamic_conf_change_nginx is defined
+
+- name: Add Dynamic config change cron
+  become: yes
+  cron:
+    name: "Dynamic conf change "
+    job: "/usr/local/bin/dynamic_restore_change_nginx --overall_restore 50 --incremental_restore 40"
+    minute: "30"
+    hour: "11"
+    user: root
+    cron_file: dynamic_conf_change_nginx
+  when: dynamic_conf_change_nginx is defined
+
+- name: Add Dynamic config change cron reset
+  become: yes
+  cron:
+    name: "Dynamic conf change "
+    job: "/usr/local/bin/dynamic_restore_change_nginx --overall_restore 25 --incremental_restore 25"
```

Comment:
can we make these values configurable in the env config?

Reply:
Made some changes. However i need help on how to iterate over zone. Can you check updated code?